gnmlibrarytool/templates/gnmlibrarytool/mediaview_storagerules.html

```diff
@@ -6,8 +6,31 @@
     <span id="mvlibtool_sr_loading">
         <img src="/sitemedia/load.gif" height="64"><span>Loading...</span>
     </span>
+    <form action="/gnmlibrarytool/delete/" method="post">
```

Comment:
One of the reasons I have done it as I have is that it needs the {% csrf_token %} to function. This is not accessible via JavaScript. It needs to live in the Django template.